config/examples/Creality/Ender-3 Pro/CrealityV422/Configuration.h

```diff
@@ -1171,7 +1170,7 @@
  * Override with M92
  *                                      X, Y, Z [, I [, J [, K...]]], E0 [, E1[, E2...]]
  */
-#define DEFAULT_AXIS_STEPS_PER_UNIT   { 80, 80, 400, 93 }
+#define DEFAULT_AXIS_STEPS_PER_UNIT   { 80, 80, 400, 111.2 }
```

Comment:
93 is the correct value for the stock extruder with no bowden tube connected. I'd leave at existing value.

Reply:
The 111.2 was the value measured on my printer, but I can set it back to 93.